rubix-common/src/main/java/com/qubole/rubix/common/metrics/BookKeeperMetrics.java

```diff
@@ -90,6 +92,19 @@ protected void initializeReporters()
           statsDReporter.start(CacheConfig.getStatsDMetricsInterval(conf), TimeUnit.MILLISECONDS);
           reporters.add(statsDReporter);
           break;
+        case GANGLIA:
+          if (!CacheConfig.isOnMaster(conf)) {
+            CacheConfig.setGangliaMetricsHost(conf, ClusterUtil.getMasterHostname(conf));
+          }
+          log.info(String.format("Reporting metrics to Ganglia [%s:%s]", CacheConfig.getGangliaMetricsHost(conf), CacheConfig.getGangliaMetricsPort(conf)));
+          final GMetric ganglia = new GMetric(CacheConfig.getGangliaMetricsHost(conf), CacheConfig.getGangliaMetricsPort(conf), GMetric.UDPAddressingMode.MULTICAST, 1);
+          final GangliaReporter gangliaReporter = GangliaReporter.forRegistry(metrics)
+                  .convertRatesTo(TimeUnit.SECONDS)
+                  .convertDurationsTo(TimeUnit.MILLISECONDS)
+                  .build(ganglia);
+          gangliaReporter.start(CacheConfig.getStatsDMetricsInterval(conf), TimeUnit.MILLISECONDS);
```

Comment:
Might be good to rename this configuration option to something more general, since we're now using it for Ganglia as well. Something like `rubix.metrics.reporting.interval` perhaps?

Reply:
Makes sense! I will rename it.